pyproject.toml

```diff
@@ -174,6 +174,9 @@ onnx-gpu = [
   "onnxruntime-gpu",
   "onnxruntime_tools",
 ]
+openai = [
+  "tiktoken>=0.1.2; python_version >= '3.8'"
+]
```

Comment:
This is not a bad idea, but I think in this case we can add `tiktoken` in the core dependencies. This way you also simplify a bit the code above, because you don't need to use `safe_import` anymore. I imagine a lot of people will be using openai embeddings in the near future and we'd like the experience to be as seamless as possible. WDYT?

Reply:
It's ok. I just didn't want to add an extra core dependency. As you authorized, I'll add as core dependency.